src/docs/pages/component-docs/components/drawer/drawer-nav-group/drawer-nav-group-doc.component.ts

```diff
@@ -65,6 +85,24 @@ export class DrawerNavGroupDocComponent {
     BASIC = BASIC;
     MULTIPLE_GROUPS = MULTIPLE_GROUPS;
     SPACER = SPACER;
+    CUSTOM_CONTENT = CUSTOM_CONTENT;
+    generatedCode: string;
+
+    /* Default playground knobs */
+    knobs: NavGroupPlaygroundKnobs = {
+        title: {
+            value: 'Title',
+            type: 'string',
+            componentDefault: '',
+            hint: 'Title that appears above nav items',
+        },
+        divider: {
+            value: true,
+            componentDefault: false,
+            type: 'boolean',
+            hint: 'Show divider under nav group',
```

Comment:
should be under the title, not under the nav group

Reply:
Updated